tests/integration/test_fim/test_registry/test_registry_report_changes/test_registry_diff_size_limit_values.py

```diff
@@ -70,50 +73,41 @@
 
 # Variables
 
-key = "HKEY_LOCAL_MACHINE"
-sub_key_1 = "SOFTWARE\\test_key"
-sub_key_2 = "SOFTWARE\\Classes\\test_key"
-
-test_regs = [os.path.join(key, sub_key_1),
-             os.path.join(key, sub_key_2)]
-
+test_regs = [os.path.join(WINDOWS_HKEY_LOCAL_MACHINE, MONITORED_KEY),
+             os.path.join(WINDOWS_HKEY_LOCAL_MACHINE, MONITORED_KEY_2)]
 test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
 wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
-reg1, reg2 = test_regs
-size_limit_configured = 10 * 1024
+size_limit_configured = SIZE_LIMIT_CONFIGURED_VALUE
```

Comment:
Use directly  SIZE_LIMIT_CONFIGURED_VALUE instead of defining a new variable size_limit_configured

Reply:
Fixed in commit https://github.com/wazuh/wazuh-qa/pull/2478/commits/8ee732a93ec3e61586e22799ba0c3bc67cab267f